src/database/migrations/0036-create-function-drop_function.sql

```diff
@@ -0,0 +1,22 @@
+-- This function came from StackOverflow which has a CC BY-SA 4.0 license
+-- https://stackoverflow.com/a/7623246/159522
```

Comment:
I'm not sure this is adequate attribution (it doesn't mention the author, for example), and I am also not sure how this licensing would interact with the rest of our codebase. If possible, let's avoid copying and pasting from StackOverflow.

Thank you for being clear about its provenance!

Reply:
I can re-write if needed, or I can just ask chat GPT to write it for me if that's better somehow!

Reply:
Fixed!